### PR TITLE
Customer can create content and only see their content.

### PR DESCRIPTION
**Pull request by simesy**

### CODE REVIEW

File: config/sync/user.role.customer.yml [hunk 1]
```diff
@@ -18,5 +21,11 @@ permissions:
   - 'create page content'
   - 'create remote_video media'
   - 'create video media'
+  - 'edit any article content'
+  - 'edit any event content'
+  - 'edit any page content'
```
Comment (tannguyen04) [inline config/sync/user.role.customer.yml:26]: I think customer role should not have this permission.

Comment (simesy) [reply config/sync/user.role.customer.yml:26]: @tannguyen04 they need this or they can't edit any pages. Workbench Access controls what they can edit.

File: config/sync/user.role.customer.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@ label: customer
 weight: 3
 is_admin: null
 permissions:
+  - 'access administration pages'
+  - 'access content overview'
```
Comment (tannguyen04) [inline config/sync/user.role.customer.yml:11]: We should not allow customers access `/admin/content`.
Customers should access workbench pages `/admin/workbench` to manage their content.

Comment (simesy) [reply config/sync/user.role.customer.yml]: Cool yep that works.

**Comment by tannguyen04** [comment]
@simesy Customers should able create content from workbench dashboard.
Screen Shot 2020-11-19 at 12 26 17
Screen Shot 2020-11-19 at 12 26 34

**Pull request merged**